cypress/integration/styling_spec.js

```diff
@@ -2,6 +2,51 @@ describe('Styling', () => {
     it("tags marked as important are bold and underlined", () => {
       cy
       .get('.important')
-      .should('have.css', 'font-weight', '700').and('have.css', 'text-decoration', 'underline')
+      .should('have.css', 'font-weight', '700').and('have.css', 'text-decoration', 'underline solid rgb(0, 0, 255)')
+    })
+    it("h3 header tags are marked as skills-header and are RebeccaPurple", () => {
+      cy
+      .get('#skills-header')
+      .should('have.css', 'color', 'rgb(102, 51, 153)')
+    })
+    it("links ending in .com are green after visited", () => {
+      cy
+      .get('a[href$=".com"]').click()
+      .should('have.css', 'color', 'rgb(0,128,0)')
+    })
+    it("tags marked as links-list have a symbol as a bullet point", () => {
+      cy
+      .get('.links-list')
+      .should('have.css', 'list-style', 'outside url("https://github.githubassets.com/images/icons/emoji/unicode/1f310.png") disc')
+    })
+    it("the last child element in all ordered lists are bold", () => {
+      cy
+      .get('ol:last-child')
```

Comment:
I think this should be `'ol li:last-child'` in order to select the last child in an ordered list, rather than the last ordered list. (At least that is what I see in the [cypress selector playground](https://docs.cypress.io/guides/core-concepts/test-runner.html#Finding-Selectors))